keyboards/candybar/info.json

```diff
@@ -1,210 +1,24 @@
-				{"label":"7",
-				"x":13,
-				"y":0},
-				{"label":"8",
-				"x":14,
-				"y":0},
-				{"label":"9",
-				"x":15,
-				"y":0},
-				{"label":"*",
-				"x":16,
-				"y":0},
-				{"label":"Tab",
-				"x":0,
-				"y":1,
-				"w":1.25},
-				{"label":"A",
-				"x":1.25,
-				"y":1},
-				{"label":"S",
-				"x":2.25,
-				"y":1},
-				{"label":"D",
-				"x":3.25,
-				"y":1},
-				{"label":"F",
-				"x":4.25,
-				"y":1},
-				{"label":"G",
-				"x":5.25,
-				"y":1},
-				{"label":"H",
-				"x":6.25,
-				"y":1},
-				{"label":"J",
-				"x":7.25,
-				"y":1},
-				{"label":"K",
-				"x":8.25,
-				"y":1},
-				{"label":"L",
-				"x":9.25,
-				"y":1},
-				{"label":":",
-				"x":10.25,
-				"y":1},
-				{"label":"Enter",
-				"x":11.25,
-				"y":1,
-				"w":1.75},
-				{"label":"4",
-				"x":13,
-				"y":1},
-				{"label":"5",
-				"x":14,
-				"y":1},
-				{"label":"6",
-				"x":15,
-				"y":1},
-				{"label":"-",
-				"x":16,
-				"y":1},
-				{"label":"Shift",
-				"x":0,
-				"y":2,
-				"w":1.75},
-				{"label":"Z",
-				"x":1.75,
-				"y":2},
-				{"label":"X",
-				"x":2.75,
-				"y":2},
-				{"label":"C",
-				"x":3.75,
-				"y":2},
-				{"label":"V",
-				"x":4.75,
-				"y":2},
-				{"label":"B",
-				"x":5.75,
-				"y":2},
-				{"label":"N",
-				"x":6.75,
-				"y":2},
-				{"label":"M",
-				"x":7.75,
-				"y":2},
-				{"label":"<",
-				"x":8.75,
-				"y":2},
-				{"label":">",
-				"x":9.75,
-				"y":2},
-				{"label":"Shift",
-				"x":10.75,
-				"y":2,
-				"w":1.25},
-				{"label":"&uarr;",
-				"x":12,
-				"y":2},
-				{"label":"1",
-				"x":13,
-				"y":2},
-				{"label":"2",
-				"x":14,
-				"y":2},
-				{"label":"3",
-				"x":15,
-				"y":2},
-				{"label":"+",
-				"x":16,
-				"y":2},
-				{"label":"Ctrl",
-				"x":0,
-				"y":3,
-				"w":1.25},
-				{"label":"GUI",
-				"x":1.25,
-				"y":3,
-				"w":1.25},
-				{"label":"Alt",
-				"x":2.5,
-				"y":3,
-				"w":1.25},
-				{"x":3.75,
-				"y":3,
-				"w":1.75},
-				{"x":5.5,
-				"y":3,
-				"w":1},
-				{"label":"Backspace",
-				"x":6.5,
-				"y":3,
-				"w":2.25},
-				{"label":"Menu",
-				"x":8.75,
-				"y":3,
-				"w":1.25},
-				{"label":"Fn",
-				"x":10,
-				"y":3},
-				{"label":"&larr;",
-				"x":11,
-				"y":3},
-				{"label":"&darr;",
-				"x":12,
-				"y":3},
-				{"label":"&rarr;",
-				"x":13,
-				"y":3},
-				{"label":"0",
-				"x":14,
-				"y":3},
-				{"label":".",
-				"x":15,
-				"y":3},
-				{"label":"Enter",
-				"x":16,
-				"y":3}]
+				{"label":"Esc", "x":0, "y":0}, {"label":"Q", "x":1, "y":0}, {"label":"W", "x":2, "y":0}, {"label":"E", "x":3, "y":0}, {"label":"R", "x":4, "y":0}, {"label":"T", "x":5, "y":0}, {"label":"Y", "x":6, "y":0}, {"label":"U", "x":7, "y":0}, {"label":"I", "x":8, "y":0}, {"label":"O", "x":9, "y":0}, {"label":"P", "x":10, "y":0}, {"label":"Del", "x":11, "y":0}, {"label":"BkSp", "x":12, "y":0}, {"label":"7", "x":13, "y":0}, {"label":"8", "x":14, "y":0}, {"label":"9", "x":15, "y":0}, {"label":"*", "x":16, "y":0},
```

Comment:
add another entry to righty and lint this file, because it doesn't seem to be legal JSON

Reply:
Done.